pages/settings/missing-assets/index.vue

```diff
@@ -60,7 +62,11 @@ const handleInput = async (customEvent: CustomEvent) => {
   assertAddress(assetAddress.value)
 
   // check if user doesn't already have the asset
-  if (connectedProfile.value?.receivedAssets?.includes(assetAddress.value)) {
+  const filteredAssetsAddresses = assetsData.value
+    ?.filter(asset => asset.isOwned)
+    .map(asset => asset.address)
+
```

Comment:
Don't we need this here as well?

```suggestion
    .map(asset => asset.address)
    .filter(asset => !!asset)
```